app/src/main/java/com/bignerdranch/android/githublogin/model/Repository.java

```diff
@@ -0,0 +1,135 @@
+package com.bignerdranch.android.githublogin.model;
```

Comment:
This class isn't needed anymore. Was actually replaced with RepositoryData, right?